docs/iis.md

```diff
@@ -13,3 +13,51 @@ The following table provides the list of metrics that the Ops Agent collects fro
 | agent.googleapis.com/iis/new_connection_count            | cumulative | 1    |        | Connections opened to IIS. |
 | agent.googleapis.com/iis/request_count                   | cumulative | 1    | state  | Requests made to IIS. |
 
+# `iis_access` Logging Receiver
+
+## Configuration
+
+To configure a receiver for your iis system logs, specify the following fields:
+
+| Field                 | Default                           | Description |
+| ---                   | ---                               | ---         |
+| `type`                | required                          | Must be `iis_access`. |
+| `include_paths`       | `[/inetpub/logs/LogFiles/W3SVC1/u_ex*]` | A list of filesystem paths to read by tailing each file. A wild card (`*`) can be used in the paths; for example, `/inetpub/logs/LogFiles/W3SVC1/u_ex*`. |
+| `exclude_paths`       | `[]`                              | A list of filesystem path patterns to exclude from the set matched by `include_paths`.
+| `wildcard_refresh_interval` | `60s` | The interval at which wildcard file paths in `include_paths` are refreshed. Specified as a time interval parsable by [time.ParseDuration](https://pkg.go.dev/time#ParseDuration). Must be a multiple of 1s.|
+
+Example Configuration:
+
+```yaml
+logging:
+  receivers:
+    iis_access:
+      type: iis_access
+  service:
+    pipelines:
+      iis:
+        receivers:
+          - iis_access
+```
+
```

Comment:
Please add a note here that it only accepts the default W3C format.

Reply:
Added